aep/general/0140/aep.md.j2

```diff
@@ -29,17 +29,31 @@ they are appropriately descriptive and of suitable length.
 
 ### Case
 
-{% tab proto %}
-
-Field definitions in protobuf files **must** use `lower_snake_case` names.
-These names are mapped to an appropriate naming convention in JSON and in
-generated code.
-
-{% tab oas %}
-
-Field definitions in OAS schemas **must** use `camelCase` names.
-
-{% endtabs %}
+JSON and protobuf fields **must** use `lower_snake_case` names. These names are
+**may** be mapped to an idiomatic naming convention in generated code.
+
+Over-the-wire references to fields in other contexts (such as query parameters,
+path segments, and embedded CEL expressions) **must** not be transformed in any
+way. The only exception to this is the rare scenario where a field name is also
+used as a header name, in which case it **must** be transformed by substituting
+hyphens (`-`) for underscores (`_`), emitted in lowercase, and parsed
+case-insensitively.
+
+**Note:** Regrettably, [ProtoJSON][proto-json] and [gRPC-Gateway][grpc-gateway]
```

Comment:
Do we need "Regrettably"?  Feels a bit editorial for the otherwise dry tone of the AEPs.

Reply:
I'm certainly not married to it, but I generally prefer to keep things a bit human? Happy to defer to consensus.